### PR TITLE
feat: add `Accelerator::key` and `Accelerator::modifiers` getters

### PR DESCRIPTION
**Pull request by stijnfrishert**
This is a PR to expose the `Modifiers` and `Code` fields within an `Accelerator`, immutably. The types are public anyway, so I don't see why you shouldn't be able to inspect what you set earlier.

I personally need this to extract the modifiers and code and fill a human readable struct with them like so. I'm only using the `Accelerator` type from `muda` because it nicely represents hotkeys and supports string parsing.

```rust
struct Hotkey {
  shift: bool,
  alt: bool,
  control: bool,
  super: bool,
  key: Code,
}
```

### CODE REVIEW

File: src/accelerator.rs [hunk 1]
```diff
@@ -98,6 +98,16 @@ impl Accelerator {
...
 
+    /// Returns the modifier for this accelerator
+    pub fn modifiers(&self) -> &Modifiers {
+        &self.mods
+    }
+
+    /// Returns the code for this accelerator
+    pub fn key(&self) -> &Code {
+        &self.key
+    }
```
Comment (amrbashir) [inline src/accelerator.rs]: `Modifiers` and `Code` are copy types so we can just return them without a reference.

```suggestion
    /// Returns the modifier for this accelerator
    pub fn modifiers(&self) -> Modifiers {
        self.mods
    }

    /// Returns the code for this accelerator
    pub fn key(&self) -> Code {
        self.key
    }
```

**Review by amrbashir — CHANGES_REQUESTED** [review]
Could you also add a change file in `.changes` directory?

**Comment by stijnfrishert** [comment]
Amended my commit and pushed the new changes, including a file in `.changes`, @amrbashir 👍

File: .changes/expose-modifiers-and-key-on-accelerator.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"muda": minor
+---
+
+Expose `modifiers()` and `key()` on `Accelerator` immutably, to be able to inspect what was set on an `Accelerator`.
```
Comment (amrbashir) [inline .changes/expose-modifiers-and-key-on-accelerator.md]:
```suggestion
Add `Accelerator::modifiers` and `Accelerator::key` getter methods.
```

File: .changes/expose-modifiers-and-key-on-accelerator.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"muda": minor
```
Comment (amrbashir) [inline .changes/expose-modifiers-and-key-on-accelerator.md]:
```suggestion
"muda": patch
```

**Review by amrbashir — APPROVED** [review]
Thank you

**Pull request merged**